.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+
+python:
+    - '3.6'
+    - '3.7'
+
+install:
+    - pip install -r requirements.txt
+    - pip install -r test_requirements.txt
+
+script:
+    - python setup.py test
```

Comment:
This is what's deprecated, should make an issue to replace this with something else.

Reply:
No, let's fix it before merge. I'll replace it with pytest instead and remove the alias in `setup.cfg`